schema/artwork/index.js

```diff
@@ -309,8 +309,37 @@ const ArtworkType = new GraphQLObjectType({
       availability: {
         type: GraphQLString,
       },
+      is_on_hold: {
+        type: GraphQLString,
+        resolve: ({ availability }) => availability === 'on hold',
+      },
+      is_not_for_sale: {
+        type: GraphQLString,
+        resolve: ({ sale_message }) => sale_message === 'Not for sale',
+      },
       sale_message: {
         type: GraphQLString,
+        args: {
+          format: {
```

Comment:
If one were to pass `sale_message(format: true)`, the response would match the spec on https://github.com/artsy/force/issues/791